File: LibreCuts_temp/app/src/main/java/com/tharunbirla/librecuts/ScratchTest.java
import com.google.android.exoplayer2.source.MergingMediaSource;
import com.google.android.exoplayer2.source.MediaSource;
public class ScratchTest {
    public void test(MediaSource s1, MediaSource s2) {
        new MergingMediaSource(true, true, s1, s2);
    }
}

File: LibreCuts_temp/app/src/main/java/com/tharunbirla/librecuts/ScratchTest.kt
package com.tharunbirla.librecuts
import com.google.android.exoplayer2.source.MergingMediaSource
import com.google.android.exoplayer2.source.MediaSource
class ScratchTest {
    fun test(s1: MediaSource, s2: MediaSource) {
        MergingMediaSource(true, s1, s2)
        MergingMediaSource(true, true, s1, s2)
    }
}
